crates/experimental-api/src/gen.rs

```diff
@@ -0,0 +1,9596 @@
+pub use gen::*;
+
+#[allow(clippy::module_inception)]
+// vvv GENERATED CODE BELOW vvv
+#[allow(dead_code)]
```

Comment:
Making this file its own lib crate would avoid the need for this and _might_ help spot issues

Reply:
Since `clippy` actually doesn't complain with `allow(dead_code)` and `allow(unused_variables)` removed, I think we're good for now.

I prefer unit-testing, that *will* spot issues :)